filebeat/input/syslog/format_check_test.go

```diff
@@ -31,3 +31,13 @@ func TestIsRFC5424(t *testing.T) {
 	assert.False(t, IsRFC5424Format([]byte("<190>2018-06-19T02:13:38.635322-0700 super mon message")))
 	assert.False(t, IsRFC5424Format([]byte("<190>589265: Feb 8 18:55:31.306: %SEC-11-IPACCESSLOGP: list 177 denied udp 10.0.0.1(53640) -> 10.100.0.1(15600), 1 packet")))
 }
+
```

Comment:
The syslog input is deprecated in 8.14 as stated in our documentation https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-syslog